### ISSUE TITLE
prepend success/failure characters on output for accessibility

### ISSUE THREAD

**Issue by cwebber** [racket/rackunit#105]
> I think I found an accessibility bug in the GUI view 🙂  It uses color indicators for test results in the tree of tests. This will work only for people who can see these colors.
> 
> Failed tests could prepend some internationalized mark for failure (maybe ✘?).

I am [proxying this bug report from the fediverse](https://mastodon.social/@sirgazil/100849726764067742).
